src/main/java/edu/nicholaidudakiwwarrick/advancedjava/model/DatabasePerson.java

```diff
@@ -0,0 +1,118 @@
+package edu.nicholaidudakiwwarrick.advancedjava.model;
+
+import javax.persistence.*;
```

Comment:
Use explicit imports not * as explicit imports are, well explicit, and the reader does not have to try to figure what is being imported from the package.